www-apps/bugzilla/bugzilla-5.0.6.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit webapp
+
+DESCRIPTION="Bugzilla is the Bug-Tracking System from the Mozilla project"
+SRC_URI="https://ftp.mozilla.org/pub/mozilla.org/webtools/${P}.tar.gz"
+HOMEPAGE="https://www.bugzilla.org"
+
+LICENSE="MPL-2.0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="mysql postgres sqlite test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="|| ( mysql postgres sqlite )"
+
+# sorting is identical to upstream MYMETA.json, please don't change
+# Theses are mandatory for checksetup.pl to configure bugzilla
+BASIC_DEPS="
+	dev-lang/perl
+	>=dev-perl/CGI-3.510.0
+	>=dev-perl/DBI-1.614.0
+	>=dev-perl/TimeDate-2.230.0
+	>=dev-perl/DateTime-0.750.0
+	>=dev-perl/DateTime-TimeZone-1.640.0
+	virtual/perl-Digest-SHA
+	>=dev-perl/Email-MIME-1.904.0
+	>=dev-perl/Email-Sender-1.300.11
+	>=dev-perl/JSON-XS-2.10.0
+	>=dev-perl/List-MoreUtils-0.320.0
+	>=dev-perl/Math-Random-ISAAC-1.0.1
+	>=dev-perl/Template-Toolkit-2.240.0
+	>=dev-perl/URI-1.550.0
+"
+
+RDEPEND="
+	${BASIC_DEPS}
+	postgres? ( >=dev-perl/DBD-Pg-1.49 )
+	mysql? ( >=dev-perl/DBD-mysql-4.0.5 )
+	sqlite? ( >=dev-perl/DBD-SQLite-1.290.0 )
+"
+need_httpd_cgi
+
+BDEPEND="
+	test? ( ${RDEPEND} )
+"
+
+PATCHES=(
```

Comment:
Submit any of these upstream if you can, even if they seem to be ignored, just in case. But otherwise, looks good to me!

Reply:
The perl-patch is a consequence of https://bugzilla.mozilla.org/show_bug.cgi?id=1588175, and I filed the two other bugs (https://bugzilla.mozilla.org/show_bug.cgi?id=1644654 and https://bugzilla.mozilla.org/show_bug.cgi?id=1644655)

Reply:
Thanks for the guide!